offload/DeviceRTL/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ set(all_amdgpu_architectures "gfx700;gfx701;gfx801;gfx803;gfx900;gfx902;gfx906"
                              "gfx908;gfx90a;gfx90c;gfx940;gfx941;gfx942;gfx1010"
                              "gfx1030;gfx1031;gfx1032;gfx1033;gfx1034;gfx1035"
                              "gfx1036;gfx1100;gfx1101;gfx1102;gfx1103;gfx1150"
-                             "gfx1151;gfx1152")
+                             "gfx1151;gfx1152;gfx1153")
```

Comment:
Not sure what happened to the indentations when adding the previous gfx11 targets…😂

Reply:
We really need to stop doing this "Compile the same runtime 50 times" thing.